Dockerfile

```diff
@@ -4,12 +4,9 @@ RUN mkdir -p /opt/pulldasher
 WORKDIR /opt/pulldasher
 
 # Install dependencies
-COPY package.json /opt/pulldasher
-RUN npm install
-
 COPY . /opt/pulldasher
+RUN npm install --unsafe-perm
```

Comment:
Instead of adding `--unsafe-perm`, can we just not run `npm install` as root?

I may be misunderstanding the situation.

Reply:
Hmmm, I think with the way the container is set up, root is the only user. I haven't added anything to create a new user just because the node app is the only thing running in the container. I'm not sure if that's an anti-pattern though.

I'll see if I can find documentation saying to do one or the other.

Reply:
I would say that inside a container, root isn't really an "elevated" privilege, right?